crates/gitbutler-core/src/project_repository/repository.rs

```diff
@@ -611,6 +612,19 @@ impl Repository {
 
         Err(RemoteError::Auth)
     }
+
+    pub fn blame(
+        &self,
+        path: &path::Path,
+        min_line: u32,
+        max_line: u32,
+        oldest_commit: &git::Oid,
+        newest_commit: &git::Oid,
+    ) -> Result<Blame, RemoteError> {
+        Ok(self
+            .git_repository
+            .blame(path, min_line, max_line, oldest_commit, newest_commit)?)
+    }
```

Comment:
We can skip having this extra layer (the same function on the project repository) - the git repository is already accessible and this just adds a layer of indirection